packages/graphiql-plugin-utils/package.json

```diff
@@ -24,14 +30,15 @@
     "cjs"
   ],
   "scripts": {
-    "build": "tsc && tsc --module CommonJS --outDir cjs"
+    "build": "tsup --dts --format esm,cjs src/index.ts"
```

Comment:
Congratulations! You finally talked less and worked more. I am proud of you 🎉

Reply:
this feels very disrespectful. I'd like you to take a break from the repo

Reply:
There is no disrespect here, just sarcasm, I don't see the reason for chatting multiple messages to just change 1 line in `package.json`

Reply:
@acao I'm really sorry about my message. It was out of place and I am very sorry having offended you.
The message clearly is offensive
I appreciate your work, our corporation and how we are working together for the past couple of months.
I'm happy to remove the message if you wish or to leave it there with my apology

Reply:
@B2o5T don't worry, we can delete the whole thread if you want, I'm sorry for being frustrating and stubborn. Enjoy your holiday!